lib/id3c/cli/redcap.py

```diff
@@ -576,6 +576,44 @@ def id(self) -> str:
             raise RuntimeError(f"Primary record id field «{self.project.record_id_field}» not available on fetched record") from e
 
 
+    def update_record(self, record: List[Dict[str, str]], date_format: str = "YMD", check_count: bool = True) -> int:
+        """
+        Updates one redcap record.
+
+        *record* must be an iterable of :py:class:``dict``s mapping REDCap field
+        names to values. The primary record id field and 
+        ``redcap_event_name`` are optional. Other pseudo-fields like
+        or ``redcap_repeat_instance`` may be necessary. All keys and
+         values must be strings.
+
+        Dates must be formatted as ``YYYY-MM-DD`` with the default
+        *date_format* of ``YMD``, as ``D/M/YYYY`` with ``DMY``, and as
+        ``M/D/YYYY`` with ``MDY``. Times are always assumed to be in the REDCap
+        server's timezone.
+
+        Any value provided for a field, including the empty string, will
+        overwrite any existing value.
+
+        This method is not suitable for creating a new record in projects that
+        use auto-numbered record ids.
+
+        Returns a count of the number of records updated, as reported by
+        REDCap which should be 1.
+        """
+        expected_count = 1
+
+        if len(record) > expected_count:
+            raise RuntimeError(f"record of lenght «{len(record)}» exceededs expected length of 1")
```

Comment:
With the reasoning above (only look at one redcap record), you do not need to check the length since you would no longer be passing in a List of Dict.